packages/output/src/browser/output-toolbar-contribution.tsx

```diff
@@ -0,0 +1,70 @@
+/********************************************************************************
+ * Copyright (C) 2019 Arm and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { inject, injectable } from 'inversify';
+import { OutputWidget } from './output-widget';
+import { OutputChannelManager } from '../common/output-channel';
+import { TabBarToolbarContribution, TabBarToolbarRegistry } from '@theia/core/lib/browser/shell/tab-bar-toolbar';
+import { OutputCommands } from './output-contribution';
+import * as React from 'react';
+
+@injectable()
+export class OutputToolbarContribution implements TabBarToolbarContribution {
+
+    @inject(OutputChannelManager)
+    protected readonly outputChannelManager: OutputChannelManager;
+
+    async registerToolbarItems(toolbarRegistry: TabBarToolbarRegistry): Promise<void> {
+        toolbarRegistry.registerItem({
+            id: 'channels',
+            render: () => this.renderChannelSelector(),
+            isVisible: widget => (widget instanceof OutputWidget),
+            onDidChange: this.outputChannelManager.onListOrSelectionChange
+        });
+
+        toolbarRegistry.registerItem({
+            id: OutputCommands.CLEAR_ALL_TOOLBAR.id,
+            command: OutputCommands.CLEAR_ALL_TOOLBAR.id,
+            tooltip: 'Clear All',
```

Comment:
Minor, I believe the tooltip should be **`Clear Output`**

Reply:
You are correct.  I have changed the tooltip as suggested.